src/Tochka.JsonRpc.Common/Tochka.JsonRpc.Common.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="JetBrains.Annotations" Version="2022.3.1" />
+    <PackageReference Include="Yoh.Text.Json.NamingPolicies" Version="1.0.0" />
```

Comment:
maybe depending on an external package with a small number of stars is a bad idea?

Reply:
This is best variant that exists. Unfortunatly System.Text.Json [doesn't support snake_case for two versions of .net already](https://github.com/dotnet/runtime/issues/782) and it will be added [only in .net8](https://github.com/dotnet/runtime/issues/71967) (I hope). This library is full copy of [PR that Microsoft accepted to System.Text.Json](https://github.com/dotnet/runtime/pull/69613) (but not released yet)